node_app/models/Appointment.js

```diff
@@ -0,0 +1,77 @@
+/*
+Appointment {
+  status : enum (proposed, pending, booked, arrived, fulfilled, cancelled, noshow, entered-in-error, checked-in, waitlist)
+  participant : [ 
+    {
+      type : "patient" // note that this type is an extensible
+      actor : string "Patient/id"
+    },
+    {
+      type : "practitioner" // note that this type is an extensible
+      actor : string "Practitioner/id"
+    },
+    {
+      type : "location" // note that this type is an extensible
+      actor : string "Location/id"
+    },
+  ]
+  slot : string "Slot/id" // note that this is not needed until scheduling is implemented
+}
+{
+  "Appointment":
+    {
+      "status":"booked",
+      "participant":[
+        {
+          "type":"patient",
+          "actor":"Patient/example"
+        },
+        {
+          "type":"pratitioner",
+          "actor":"Practitioner/f201"
+        },
+        {
+          "type":"patient",
+          "actor":"Patient/102"
```

Comment:
Is this a typo?  You have "patient" on lines 26/27 and 34/35

Reply:
Thanks. Fixed it.